sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2020,6 +2020,15 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 						"ALTER TABLE eventParticipants ADD COLUMN sormasToSormasOriginInfo_id bigint REFERENCES sormasToSormasOriginInfo(id);");
 				getDao(EventParticipant.class).executeRaw("ALTER TABLE eventParticipants ADD COLUMN ownershipHandedOver boolean;");
 
+			case 279:
```

Comment:
Needs to be 280, and the version number at the top of the class needs to be updated to 281